api/src/migrations/api/1611329857558-initialDataMetadataEntities.ts

```diff
@@ -0,0 +1,89 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class initialDataMetadataEntities1611329857558 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query(`
+      CREATE TYPE "status" AS ENUM (
+        'created',
+        'running',
+        'done',
+        'failure'
+      );
+
+      CREATE TYPE "tags" AS ENUM (
```

Comment:
maybe same as above, though this could be generic enough. though we may over time have different kinds of tags for different kinds of entities. again, without overthinking this now :)